### ISSUE TITLE
Day 2 Agenda

### ISSUE THREAD

**Issue by dykesb** [gvsu-sta518/community#2]
1. Grades Discussion
2. Project Discussion
3. GitHub Collaboration

**Comment by LeviRosendall** [comment]
The muddiest thing for me is how are we going to be able to find each other's code, or will we publish all of our issues in this community?

**Comment by AverySmithson** [comment]
The muddiest thing for me was the video about forks and pull requests (video 3).

**Comment by keydels** [comment]
The muddiest thing for me was creating a pull request without getting an error?

**Comment by ian-curtis** [comment]
I have experience with Git and Github so a lot of this was review. But something that has always been muddy for me is managing merge conflicts and how to determine whose commit should become part of the `main` branch.

**Comment by lcadagin** [comment]
The presenter was very careful not to confuse "forking" with "cloning".  My question would be what "cloning" is in GitHub and how it differs from "forking"?

**Comment by ian-curtis** [comment]
> The presenter was very careful not to confuse "forking" with "cloning". My question would be what "cloning" is in GitHub and how it differs from "forking"?

I can answer this one (at least partly)! "Cloning" is the name normally given to getting access to the files in a repository on your personal computer so you can edit them in a coding software (and don't have to use Github directly as we have been). In that way, you can test code and try different things. Cloning (in my experience) involves using Git with Github which hasn't been covered yet in videos so I wouldn't be too concerned with it for the time being.

In short, "Forking" copies a repo on Github to your personal account, "cloning" copies a repo "locally" or adds files to your own computer. Hope this makes sense for now!

**Comment by manapradit** [comment]
The GitHub version in the video and my screen are slightly different, i.e. graphs tab to view the branches network. The video was from 2016.

**Comment by TylerToren** [comment]
I am still a little confused about what the difference is between a push request and a pull request.

**Comment by mahmisab** [comment]
I'm new to GitHub. I'm not sure if I added the gif correctly. Will definitely take a hot sec to get used to GitHub.

**Comment by mahmisab** [comment]
> I'm new to GitHub. I'm not sure if I added the gif correctly. Will definitely take a hot sec to get used to GitHub.

Wait, fixed the gif issue!

**Comment by elaynast** [comment]
I really don't understand what the README file is or what merging it does. The first time I tried to make a pull request for my readme file it didn't work. I deleted my repository and tried again and it worked the second time, but I'm really not sure why.

**Comment by proctorlauren** [comment]
Almost everything I was muddy on was previously mentioned! One thing I would add is that I am still muddy on the markdown tools and how to format everything. I bookmarked the markdown tutorial so I can go back and reference it if I am confused!

**Comment by dahalkush** [comment]
The muddiest thing for me was i was trying to commit the changes in Readme.md file by adding the GIF link inside the new branch: Blue but it did not work. Due to which there was no change in main branch and the blue one so could not create a pull request between them.

**Comment by nash-meg** [comment]
The muddiest thing for me was finding the issues for this community. For some reason I kept getting the spring/summer github page for this class, so it took me a second to find our fall class (and I'm still not entirely sure why I kept ending up on the other one).

**Comment by damosack** [comment]
I'm confused about how to find other repositories easily. I'm sure I was doing something wrong, but when I was looking for this one I had to resort to just using the link to the preparation01 tasks and going backwards to the repository it's in.

**Comment by lindsaybandyk** [comment]
> The muddiest thing for me was creating a pull request without getting an error?

i continually got an error when trying to pull my main branch with my gif

**Comment by SSterling7** [comment]
Muddiest thing was making the initial markdown file because im blind apparently. In all seriousness I feel okay right now, just that I need to do this like 5 billion times before I feel comfortable moving around the site.

**Comment by debit7** [comment]
The muddiest thing is that , we never used to do anything in readme.md file in our project in office.

**Comment by debit7** [comment]
The muddiest thing is that , we never used to do anything in readme.md file in our project in office.
